### ISSUE TITLE
Build configurations for different rendering engines

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#15632]
Do we, on Jenkins, have a way/script to compile a build which for rendering engine _Version 1_ supports "_fat_" (armeabi-v7a, arm64-v8a, x86, x86_64), and for _Version 2_ just _arm64-v8a_?

I am looking to build a test build I can hand to members of an expedition for practical testing, but do not know their devices, want them to be able to fall back to Version 1 in any case.

**Comment by vshcherb** [comment]
Yes, we ```assembleAndroidFullOpenglFatDebug``` (OpenGl) ```assembleAndroidFullLegacyFatDebug``` (smaller no opengl).
We need configuration as well to "exclude " mini basemap probably
